### `ERR_FS_CP_SOCKET`

<!--
added: v16.7.0
-->

An attempt was made to copy to a socket with [`fs.cp()`][].

<a id="ERR_FS_CP_SYMLINK_TO_SUBDIRECTORY"></a>
